StockPriceScraper/Program.cs

```diff
@@ -20,18 +20,18 @@ class Program
     {
 
         // Items from the configuration file
-        private static string stockBaseUrl = "";
-        private static string eurXpath = "";
-        private static string usdXpath = "";
-        private static string outputfilePath = "";
-        private static string stockNameXpath = "";
-        private static string wknXpath;
+        private static string stockBaseUrl = string.Empty;
+        private static string eurXpath = string.Empty;
+        private static string usdXpath = string.Empty;
+        private static string outputfilePath = string.Empty;
+        private static string stockNameXpath = string.Empty;
+        private static string wknXpath = string.Empty;
 
         // Array with stock Names
-        static List<string> stockNames = new List<string>();
+        private static readonly List<string> stockNames = new();
```

Comment:
Why readonly?

Reply:
Because you are not assigning a new List<string> ever to this variable. This variable will always point to the same list. What you do is adding values to the list.